srcpkgs/pdfarranger/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'pdfarranger'
+pkgname=pdfarranger
+version=1.7.1
+revision=1
+build_style=python3-module
+hostmakedepends="
+ ghostscript intltool python3-setuptools python3-distutils-extra python3-pikepdf
+ python3-pip python3-xlib python3-wheel"
+makedepends="
+ ghostscript-devel poppler-glib-devel python3-cairo-devel python3-gobject-devel
+ gtk+3-devel"
+depends="
+ gtk+3 python3-cairo python3-gobject python3-pikepdf python3-dateutil
+ ghostscript python3-setuptools python3-xlib poppler-glib img2pdf"
+checkdepends="python3-wheel python3-pip $depends"
```

Comment:
Can the checks work without pip? With pip they are useless and should be disabled.

Reply:
oke. i have deleted.